### PR TITLE
added some external projects

### PR DESCRIPTION
**Pull request by wanko**

### CODE REVIEW

File: krr.bib [hunk 1]
```diff
@@ -20931,3 +20931,20 @@ @Article{brkaryxiza18a
   volume =	 62,
   pages =	 "829-877"
 }
+
+@InProceedings{phalmi19a,
+  author =  "T. Pham and M. Ali and A. Mileo",
+  title =  "C-ASP: Continuous ASP-Based Reasoning over RDF Streams",
```
Comment (tortinator) [inline krr.bib:20937]:
```suggestion
  title =  "{C-ASP}: Continuous {ASP}-Based Reasoning over {RDF} Streams",
```

File: krr.bib [hunk 2]
```diff
@@ -20931,3 +20931,20 @@ @Article{brkaryxiza18a
...
+  year =  "2019",
+  crossref =	 "lpnmr19"
+}
+
+@inbook{almara18a,
```
Comment (tortinator) [inline krr.bib:20942]:
```suggestion
@InBook{almara18a,
```

File: krr.bib [hunk 3]
```diff
@@ -20931,3 +20931,20 @@ @Article{brkaryxiza18a
...
+  author = "F. Al Machot and H. Mayr and S. Ranasinghe",
```
Comment (tortinator) [inline krr.bib:20943]:
```suggestion
  author = "F. {Al Machot} and H. Mayr and S. Ranasinghe",
```

File: krr.bib [hunk 4]
```diff
@@ -20931,3 +20931,20 @@ @Article{brkaryxiza18a
...
+  year = "2018",
+  pages = "303-318",
+  title = "A Hybrid Reasoning Approach for Activity Recognition Based on Answer Set Programming and Dempster–Shafer Theory",
+  isbn = "978-3-319-58995-4",
```
Comment (tortinator) [inline krr.bib:20947]:
```suggestion
```

Delete

File: krr.bib [hunk 5]
```diff
@@ -20931,3 +20931,20 @@ @Article{brkaryxiza18a
...
+  book = "Studies in Systems, Decision and Control",
+  doi = "10.1007/978-3-319-58996-1_14"
```
Comment (tortinator) [inline krr.bib:20949]:
```suggestion
  book = "Studies in Systems, Decision and Control"
```

Delete

**Comment by wanko** [comment]
Thanks for teaching me the details, ;)

**Comment by tortinator** [comment]
Sorry, wasn't meant as oberlehrerisch :)

**Comment by wanko** [comment]
> 
> 
> Sorry, wasn't meant as oberlehrerisch :)

Not at all. :D I am seriosly bad at spotting those.

**Comment by tortinator** [comment]
Ok, ready for more? O:)


@InCollection{almara18a,
  author = "F. {Al Machot} and H. Mayr and S. Ranasinghe",
  title = "A Hybrid Reasoning Approach for Activity Recognition Based on
Answer Set Programming and Dempster–Shafer Theory",
  booktitle = "Recent Advances in Nonlinear Dynamics and Synchronization",
  publisher = springer,
  year = 2018,
  editor = "K. Kyamakya and W. Mathis and R. Stoop and J. Chedjou and Z.
Li",
  volume = 109,
  series = "Studies in Systems, Decision and Control",
  pages = "303-318"
}

**Pull request merged**